data/json/monstergroups/zombies.json

```diff
@@ -317,7 +317,8 @@
     "default": "mon_zombie_grappler",
     "monsters": [
       { "monster": "mon_zombie_grappler", "freq": 40, "cost_multiplier": 0 },
-      { "monster": "mon_zombie_biter", "freq": 40, "cost_multiplier": 2 }
+      { "monster": "mon_zombie_biter", "freq": 40, "cost_multiplier": 2 },
+      { "monster": "mon_frog_mother", "freq": 2, "cost_multiplier": 4 }
```

Comment:
I think this may be the evolution group for grabber zombies, you might not want to add it here.